### ISSUE TITLE
When i try to create a new oppurtunity i can select only the first 25 accounts

### ISSUE THREAD

**Issue by thomasrossetto** [fatfreecrm/fat_free_crm#589]
Hi, 
when i try to create a new opportunity, and i want select an existing account, i can select only the first 25 accounts. I wrote a test to show it.

```
scenario 'should create a new opportunity with a lot of account', js: true do
   1.upto(40) { |i| FactoryGirl.create(:account, name: "Account #{i.to_s.rjust(2, '0')}") }
    with_versioning do
      visit opportunities_page
      click_link 'Create Opportunity'
      expect(page).to have_selector('#opportunity_name', visible: true)
      fill_in 'opportunity_name', with: 'My Awesome Opportunity'
      click_link 'select existing'
      expect(page).to have_content('Account 25')
      expect(page).to have_content('Account 26')
      chosen_select('Account 30', from: 'account_id')
      click_link 'Comment'
      fill_in 'comment_body', with: 'This is a very important opportunity.'
      click_button 'Create Opportunity'
      expect(page).to have_content('My Awesome Opportunity')
      find('div#opportunities').click_link('My Awesome Opportunity')
      expect(page).to have_content('This is a very important opportunity.')
      click_link "Dashboard"
      expect(page).to have_content("Bill Murray created opportunity My Awesome Opportunity")
      expect(page).to have_content("Bill Murray created comment on My Awesome Opportunity")
    end
  end
```

How can i do to show all accounts ? Otherwise, how can i do to search in all accounts and not only in 25 ? 

Thanks for help!

**Comment by CloCkWeRX** [comment]
This is a bit tricky if you have (like we do) 10,000+ accounts - performance really suffers.

We should probably swap to an autocomplete or similar.

**Comment by johnnyshields** [comment]
@CloCkWeRX we need autocomplete with a query that is properly indexed and limited. It's not hard to do, performance won't suffer. This is a priority to add.

**Comment by CloCkWeRX** [comment]
Actually... isn't this already an autocomplete?

<img width="729" alt="screen shot 2018-01-21 at 12 28 25 am" src="https://user-images.githubusercontent.com/365751/35184203-22736648-fe42-11e7-8459-6226c7062855.png">

**Comment by johnnyshields** [comment]
No, it doesn't fetch new results from the server, at least the one in "Create Opportunity". Open Network tab in chrome and there's no traffic.

**Comment by CloCkWeRX** [comment]
<img width="1680" alt="screen shot 2018-01-21 at 12 38 45 am" src="https://user-images.githubusercontent.com/365751/35184284-8b76e704-fe43-11e7-8235-3076038a6477.png"> 
?

unless we forked it

**Comment by CloCkWeRX** [comment]
This should be fixed via #724 in the upcoming 0.17.2 release; and working since 0.14.0-0.16.0
Going to skip the test coverage for the moment
